src/phy/mod.rs

```diff
@@ -179,6 +179,7 @@ pub struct ChecksumCapabilities {
     pub udpv4: Checksum,
     pub tcpv4: Checksum,
     pub icmpv4: Checksum,
+    pub igmp: Checksum,
```

Comment:
I don't think there's any NIC that does IGMP checksum offload. Even ICMP was a surprise to me.

Reply:
Fixed